build/Makefile

```diff
@@ -512,12 +497,14 @@ endif
 #--------------------------------------------------------------------------------
 # Run unit tests
 #
+# Note: with the upgrade to ruby 2.3, rake needs the temporary PATH change to
```

Comment:
@jeffaco I still think this case is special to highlight because it is changing system environment variables. devs can easily confuse the $PATH local parameter used in make vs system configuration. And this workaround is needed for only 2.3 (was not a case in 2.2)